Neos.Media/Classes/Command/MediaCommandController.php

```diff
@@ -127,6 +128,55 @@ public function importResourcesCommand($simulate = false)
     }
 
     /**
+     * Remove unused assets
+     *
+     * This command iterates over all existing assets, checks their usage count
+     * and lists the assets which are not reported as used by any AssetUsageStrategies.
+     * The unused assets can than be removed.
+     */
+    public function removeUnusedCommand()
+    {
+        $iterator = $this->assetRepository->findAllIterator();
+        $assetCount = $this->assetRepository->countAll();
+        $unusedAssets = [];
+        $unusedAssetInfo = [];
+        $unusedAssetCount = 0;
+
+        $this->outputLine('<b>Searching for unused assets:</b>');
+
+        $this->output->progressStart($assetCount);
+        /** @var AssetInterface $asset */
+        foreach ($this->assetRepository->iterate($iterator) as $asset) {
+            if ($asset->getUsageCount() === 0) {
+                $unusedAssets[] = $asset;
+                $unusedAssetInfo[] = sprintf('- %s (%s)', $asset->getIdentifier(), $asset->getResource()->getFilename());
+                $unusedAssetCount++;
+            }
+            $this->output->progressAdvance(1);
+        }
+
+        if ($unusedAssetCount === 0) {
+            $this->output->outputLine(PHP_EOL . sprintf('No unused assets found.', $unusedAssetCount));
+            $this->quit(0);
+        }
+
+        $this->outputLine(PHP_EOL . 'Found the following unused assets: ' . PHP_EOL . implode(PHP_EOL, $unusedAssetInfo));
+        $this->output->outputLine(sprintf('Found <b>%s</b> unused assets.', $unusedAssetCount));
```

Comment:
sorry for nit picking, but isn't this line a bit redundant? the first line could just say how many unused were found, no? or is it in case so many were found that it wouldn't be visible? if so then I'd suggest to add it to the question, "Do you want to remove these 3 unused assets?"